lntest/harness.go

```diff
@@ -249,19 +255,20 @@ out:
 
 // TearDownAll tears down all active nodes within the test lightning network.
 func (n *NetworkHarness) TearDownAll() error {
```

Comment:
Should we just call this `ShutdownNodes` now?

Reply:
Renamed it to `TearDown()` as I like the `SetUp()` and `TearDown()` convention that is also used in other test frameworks case scope harnesses.